src/engines/nwn2/roster.cpp

```diff
@@ -65,6 +65,29 @@ Common::UString Roster::getNextRosterMember() {
 	return (it != _members.end()) ? it->rosterName : "";
 }
 
+bool Roster::getIsRosterMemberAvailable(Common::UString name) const {
+	std::list<Member>::const_iterator it = getRosterMember(name);
+	return (it != _members.end()) ? it->isAvailable : false;
+}
+
+bool Roster::getIsRosterMemberCampaignNPC(Common::UString name) const {
+	std::list<Member>::const_iterator it = getRosterMember(name);
+	return (it != _members.end()) ? it->isCampaignNPC : false;
+}
+
+bool Roster::getIsRosterMemberSelectable(Common::UString name) const {
+	std::list<Member>::const_iterator it = getRosterMember(name);
+	return (it != _members.end()) ? it->isSelectable : false;
+}
+
+std::list<Roster::Member>::const_iterator Roster::getRosterMember(Common::UString name) const {
```

Comment:
xoreos has been opened for C++11, so you could theoretically use std::find() (in `<algorithm>`) with a lambda here. It's not necessary, though.

Reply:
I'll have to revisit this when I do the 'set' calls, so I can experiment then.